README.md

```diff
@@ -17,11 +17,11 @@ Inputs
 ------
 
 - `vswhere`: Path to `vswhere.exe` (default system-installed copy).
-- `arch`: Build architecture (default `amd64`).
-- `host_arch`: Host architecture override.
+- `host_arch`: Host architecture override (defaults to the processor architecture).
+- `arch`: Build architecture (defaults to the value of host_arch).
```

Comment:
This is the host architecture not the build architecture right? This is where the generated binaries will run.

Reply:
See previous comment